nist_csf_v2/function_id.pp

```diff
@@ -0,0 +1,277 @@
+  children = [
+    benchmark.nist_csf_v2_id_am_01,
+    benchmark.nist_csf_v2_id_am_02,
+    benchmark.nist_csf_v2_id_am_03,
+    benchmark.nist_csf_v2_id_am_05,
+    benchmark.nist_csf_v2_id_am_07,
+    benchmark.nist_csf_v2_id_am_08
+  ]
+  tags = local.nist_csf_v2_common_tags
+}
+
+benchmark "nist_csf_v2_id_ra" {
+  title       = "Risk Assessment (ID.RA)"
+  description = "The cybersecurity risk to the organization, assets, and individuals is understood by the organization."
+  children = [
+    benchmark.nist_csf_v2_id_ra_01,
+    benchmark.nist_csf_v2_id_ra_03,
+    benchmark.nist_csf_v2_id_ra_04,
+    benchmark.nist_csf_v2_id_ra_05,
+    benchmark.nist_csf_v2_id_ra_08,
+    benchmark.nist_csf_v2_id_ra_09
+  ]
+  tags = local.nist_csf_v2_common_tags
+}
+
+benchmark "nist_csf_v2_id_im" {
+  title       = "Improvement (ID.IM)"
+  description = "Improvements to organizational cybersecurity risk management processes, procedures and activities are identified across all CSF Functions."
+  children = [
+    benchmark.nist_csf_v2_id_im_01,
+    benchmark.nist_csf_v2_id_im_02
+  ]
+  tags = local.nist_csf_v2_common_tags
+}
+
+benchmark "nist_csf_v2_id_am_01" {
+  title       = "ID.AM-01"
+  description = "Inventories of hardware managed by the organization are maintained."
+  children = [
+    control.ec2_instance_detailed_monitoring_enabled,
+    control.ec2_instance_ssm_managed,
+    control.ec2_stopped_instance_30_days,
+    control.ssm_managed_instance_compliance_association_compliant
+  ]
+  tags = local.nist_csf_v2_common_tags
+}
+
+benchmark "nist_csf_v2_id_am_02" {
+  title       = "ID.AM-02"
+  description = "Inventories of software, services, and systems managed by the organization are maintained."
+  children = [
+    control.config_enabled_all_regions,
+    control.ec2_instance_ssm_managed,
+    control.ssm_managed_instance_compliance_association_compliant
+  ]
+  tags = local.nist_csf_v2_common_tags
+}
+
+benchmark "nist_csf_v2_id_am_03" {
+  title       = "ID.AM-03"
+  description = "Representations of the organization's authorized network communication and internal and external network data flows are maintained."
+  children = [
+    control.cloudtrail_multi_region_trail_enabled,
+    control.cloudtrail_trail_enabled,
+    control.elb_application_classic_lb_logging_enabled,
+    control.log_metric_filter_network_acl,
+    control.log_metric_filter_network_gateway,
+    control.log_metric_filter_route_table,
+    control.log_metric_filter_security_group,
+    control.log_metric_filter_vpc,
+    control.s3_bucket_logging_enabled,
+    control.vpc_eip_associated,
+    control.vpc_flow_logs_enabled,
+    control.vpc_network_acl_unused
+  ]
+  tags = local.nist_csf_v2_common_tags
+}
+
+benchmark "nist_csf_v2_id_am_05" {
+  title       = "ID.AM-05"
+  description = "Assets are prioritized based on classification, criticality, resources, and impact on the mission."
+  children = [
+    control.backup_plan_min_retention_35_days,
```

Comment:
Shall we have backup controls under `ID.AM-05` as the benchmark highlights on assets prioritization based on classification, criticality?
Please can you take another look at whether the controls are placed under proper benchmark